oaknational/Oak-Web-Application
# feat: remove the isSensitive units and lessons from pupil browse

## Description

Music year: 2005

Removed any units or lessons with the isSensitive field set to `true`.
This has been done within getStaticProps so that it happens server side.

## How to test

1. Go to https://deploy-preview-2447--oak-web-application.netlify.thenational.academy/pupils/beta/programmes/rshe-pshe-secondary-year-10-l/units
2. You should see 4 units (before there were 6)
3. Go to https://deploy-preview-2447--oak-web-application.netlify.thenational.academy/pupils/beta/programmes/rshe-pshe-secondary-year-10-l/units/online-and-media-dangers-of-viewing-explicit-material-6a7c/lessons
4. You should see 4 lessons (before there were 5)

## Screenshots

| Current | New |
|----------|----------|
| <img width="1324" alt="image" src="https://github.com/oaknational/Oak-Web-Application/assets/587241/0068efe5-4113-4cf6-aa82-364a9686c807"> | <img width="1245" alt="image" src="https://github.com/oaknational/Oak-Web-Application/assets/587241/7be8d25f-d0d1-42ac-9883-3ba0170fdfbf"> |

## Checklist
